README.md

```diff
@@ -611,8 +662,59 @@ cc_cold_xsk1fp4megtpn4vu4cug2lmsyhg4xvnnar55q6k8wp5e6f2h8jz849ph8v8jhm0qffw8v6ut
 $ cardano-address key public --with-chain-code < cold.xsk > cold.xvk
 cc_cold_xvk1dg8d5du0v4ukqkfgset50xncudhwlfzz2p6epv096x0ndl8jsgzzdqzpe9yw37u7mu7xaenhv7242990ps3sn8jcg52yx7n3fuke9kst5t2py
 
-$ cardano-address key hash < cold.xvk > cold
+$ cardano-address key public --without-chain-code < cold.xsk > cold.vk
+cc_cold_vk1dg8d5du0v4ukqkfgset50xncudhwlfzz2p6epv096x0ndl8jsgzqmwj2x5
+
+$ cardano-address key hash < cold.xvk > cold.vkh
+cc_cold_vkh1d7yw362prvnae5fc8063xdeapws9ptzdgjkqd4dk3qddctn5rch
+
+$ cat cold.vkh  | bech32
+6f88e8e9411b27dcd1383bf513373d0ba050ac4d44ac06d5b6881adc
+
+$ cat cold.vkh  | bech32 cc_cold > cold.deprecated
 cc_cold1d7yw362prvnae5fc8063xdeapws9ptzdgjkqd4dk3qddccyzfjm
+
+$ cat cold.deprecated | bech32
+6f88e8e9411b27dcd1383bf513373d0ba050ac4d44ac06d5b6881adc
+
+$ cardano-address key hash --with-byte < cold.xvk > cold.credential
+cc_cold1zfhc368fgydj0hx38qal2yeh8596q59vf4z2cpk4k6yp4hqy3mpsx
+
+$ cat cold.credential | bech32
+126f88e8e9411b27dcd1383bf513373d0ba050ac4d44ac06d5b6881adc
+
+(there is the expected 0x12 prepended byte as it is cc cold key hash credential in accordance to CIP-0129. The corresponding key hash is '6f88e8e9411b27dcd1383bf513373d0ba050ac4d44ac06d5b6881adc' and it is the same as in case of both `cold.vkh` and `cold.deprecated`).
+```
+</details>
+
+<details>
+  <summary>How to create script and its hash from cold committee keys (<strong>drep</strong>)</summary>
+
+```console
+
+$ cat cold.vkh
+cc_cold_vkh1d7yw362prvnae5fc8063xdeapws9ptzdgjkqd4dk3qddctn5rch
+$ cat cold.deprecated
+cold1d7yw362prvnae5fc8063xdeapws9ptzdgjkqd4dk3qddcj20wc6
```

Comment:
This is the wrong prefix, for CIP105 it should be `cc_cold`

so this should be 
`cc_cold1d7yw362prvnae5fc8063xdeapws9ptzdgjkqd4dk3qddccyzfjm`

Reply:
good catch, thanks, fixed